docs/spec/wire-format.md

```
@@ -155,3 +155,14 @@ LK-SHA256: 9a7c…
 - `schemas/policy_result.schema.json` — Policy evaluation result
 
 These schemas are informative; the canonicalization and hashing rules above are normative.
+### CBOR Canonical Profile (Optional)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)